packages/react/src/components/Tile/next/Tile.js

```diff
@@ -0,0 +1,692 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { Component, useState } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Link from '../../Link';
+import {
+  Checkbox16,
+  CheckboxCheckedFilled16,
+  ChevronDown16,
+} from '@carbon/icons-react';
+import { keys, matches } from '../../../internal/keyboard';
+import deprecate from '../../../prop-types/deprecate';
+import { composeEventHandlers } from '../../../tools/events';
+import { PrefixContext, usePrefix } from '../../../internal/usePrefix';
+
+export const Tile = React.forwardRef(function Tile(
+  { children, className, light = false, ...rest },
+  ref
+) {
+  const prefix = usePrefix();
+
+  const tileClasses = cx(
+    `${prefix}--tile`,
+    {
+      [`${prefix}--tile--light`]: light,
+    },
+    className
+  );
+  return (
+    <div className={tileClasses} ref={ref} {...rest}>
+      {children}
+    </div>
+  );
+});
+
+Tile.displayName = 'Tile';
+Tile.propTypes = {
+  /**
+   * The child nodes.
+   */
+  children: PropTypes.node,
+
+  /**
+   * The CSS class names.
+   */
+  className: PropTypes.string,
+
+  /**
```

Comment:
same as other comment - we should add a deprecation warning here

Reply:
On className?

Reply:
weird, I thought I highlighted `light` prop 🤔  No, I'm talking about all of the `light` props.

Reply:
kewl